FreeRTOS-Plus/Demo/FreeRTOS_Plus_TCP_Echo_Posix/main_networking.c

```diff
@@ -118,6 +118,18 @@ const uint8_t ucMACAddress[ 6 ] = { configMAC_ADDR0, configMAC_ADDR1, configMAC_
 /* Use by the pseudo random number generator. */
 static UBaseType_t ulNextRand;
 
+#if ( ipconfigMULTI_INTERFACE == 1 ) && ( ipconfigCOMPATIBLE_WITH_SINGLE == 0 )
```

Comment:
Ideally, we do not want multiple macros. This part of the code also has endpoint changes like "static NetworkEndPoint_t xEndPoints[4];", so using a single macro makes more sense. We can use FREERTOS_PLUS_TCP_VERSION in all the places.

Reply:
I basically want to understand the use case where adding 2 extra macros ipconfigMULTI_INTERFACE and ipconfigCOMPATIBLE_WITH_SINGLE will make the demo more configurable or more useful, otherwise adding extra macros is only going to add up to more maintainability related issues. From the new version of FREERTOS_PLUS_TCP_VERSION >= 10, it is always multiple interface by default, so it makes more sense to have a single macro.